scripts/python/mod_ac_loadout_puller.py

```diff
@@ -0,0 +1,69 @@
+import os
+
+# save DCS.openbeta\\mods\\aircraft path as variable (you will need to put in the path to your own directory)
+path = "C:\\Users\\USERNAME\\Saved Games\\DCS.openbeta\\Mods\\aircraft"

```

Comment:
```python
path = f"{os.environ['UserProfile']}\\Saved Games\\DCS.openbeta\\Mods\\aircraft"
```
Can be used as default to fetch from current user's saved games folder

Reply:
thanks! I had a brain fart yesterday and went looking how to do this via some wildcard within pathlib/glob and completely forgot about os.environ!